README.md

```diff
@@ -18,17 +18,48 @@ import Lightbox from 'react-images';
 
 export default class Sample extends React.Component {
   ...
+
+  state = {
+    currentImage: 0,
+    lightboxIsOpen: false,
+    images: [{ src: 'http://example.com/img1.jpg' }, { src: 'http://example.com/img2.jpg' }]
+  }
+
   render() {
+    const { currentImage, images, lightboxIsOpen } = this.state
     return (
-      <Lightbox
-        images={[{ src: 'http://example.com/img1.jpg' }, { src: 'http://example.com/img2.jpg' }]}
-        isOpen={this.state.lightboxIsOpen}
-        onClickPrev={this.gotoPrevious}
-        onClickNext={this.gotoNext}
-        onClose={this.closeLightbox}
-      />
+      <div>
+        <button onClick={this.onOpen}>Open</button>
+        <Lightbox
+          currentImage: currentImage
+          images={images}
+          isOpen={lightboxIsOpen}
+          onClickPrev={this.gotoPrevious}
+          onClickNext={this.gotoNext}
+          onClose={this.onClose}
+        />
+      </div>
     );
   }
+
+  onOpen = evt => {
```

Comment:
Please use `myMethod () {}` syntax for:
- onOpen
- onClose
- onClickPrev
- onClickNext